sdk/mobile-center-rum/proguard-rules.pro

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
+# By default, the flags in this file are appended to flags specified
+# in C:\Users\ivan.matkov\AppData\Local\Android\Sdk/tools/proguard/proguard-android.txt
```

Comment:
I guess we should remove our local paths even from comments.
Maybe it's better to use the `proguard` file, as in `analytics` module?

Reply:
cleanup should be done on develop then for the other ones. I can clean this one here.